assets/stylesheets/components/_app-header.scss

```diff
@@ -0,0 +1,7 @@
+.app-header {
+  @apply max-w-md w-full;
```

Comment:
Ditto here: these styles should go in the layout. But the title part can be in the component.